src/Iot.Device.Bindings/Directory.Build.props

```diff
@@ -1,8 +1,8 @@
 <Project>
   <!-- Packaging related properties -->
   <PropertyGroup>
-    <MajorVersion>1</MajorVersion>
-    <MinorVersion>6</MinorVersion>
+    <MajorVersion>2</MajorVersion>
```

Comment:
Should `PackageValidationBaselineVersion` use `1.6.0`?

Reply:
Good point, 1.6.0 never shipped but we should probably be using 1.5.0. In any case, I don't want to make those changes as part of this PR but I can send a subsequent for that part. FWIW we did run package validation against 1.5.0 baseline manually and found only expected changes in there, so I'm not too worried. That said, we should definitely change this  in case there is anything that needs to be fixed before 2.0 RTMs